src/workspacer.Shared/IConfigContext.cs

```diff
@@ -23,7 +23,12 @@ public interface IConfigContext
 
         IMonitorContainer MonitorContainer { get; set; }
         bool CanMinimizeWindows { get; set; }
-        
+        WindowOrder NewWindowOrder { get; set; }
+        public enum WindowOrder
+        {
+            NewWindowsLast, // default
+            NewWindowsFirst
+        }
```

Comment:
It would be better if this enum was a standalone entity and not nested within another.

Reply:
did it 👍